data/logs.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "title": "titulo",
+    "content": "conteúdo",
+    "datePublished": "2077-10-02 9:00",
+    "forDisplayTime": "9H00",
+    "path": "/logs"
+  },
+  {
+    "title": "título2",
+    "content": "conteúdo2",
+    "datePublished": "2023-08-02 21:40",
+    "forDisplayTime": "21H40",
+    "path": "/logs"
+  },
+  {
+    "title": "título3",
+    "content": "conteúdo3",
+    "datePublished": "2023-08-02 21:37",
+    "forDisplayTime": "21H37",
+    "path": "/"
+  }
```

Comment:
In your json, you don't need two values to represent the same date. Keep using just the `datePublished` value and remove the `forDisplayTime`. In the code, you can format your date with some **date utility library**, such as `date-fns` or `dayjs`. I'd suggest **dayjs** because of it's package size. 

And then, your code would become something like: 

```tsx
<time dateTime={new Date(datePublished).toISOString()}>{dayjs(datePublished).format("HH'H'mm")}</time>
```

Reply:
Already changed it, thank you!